### PR TITLE
Move scripts to scripts folder and update readme

### PR DESCRIPTION
**Pull request by xlc**
I move init.sh and build.sh to scripts to make it align with substrate so that the instructions are same.
Copied some relevant part from the main readme file.

### CODE REVIEW

File: node-template/README.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+Install required tools:
+
+```bash
+./scripts/init.sh
+```
+
+Build the WebAssembly binary
+
+```bash
+./scripts/init.sh
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
./scripts/build.sh
```

File: node-template/README.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+curl https://sh.rustup.rs -sSf | sh
+```
+
+Install required tools:
+
+```bash
+./scripts/init.sh
+```
+
+Build the WebAssembly binary
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
Build the WebAssembly binary:
```

File: node-template/README.md [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+
+```bash
+./scripts/init.sh
+```
+
+Build all native code
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
Build all native code:
```

File: node-template/README.md [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+
+You can start a development chain with:
+
+```bash
+cargo run -- --dev
+```
+
+Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- --dev`.
+
+If you want to see the multi-node consensus algorithm in action locally, then you can create a local testnet with two validator nodes for Alice and Bob, who are the initial authorities of the genesis chain specification that have been endowed with a testnet DOTs. We'll give each node a name and expose them so they are listed on link: [Telemetry](https://telemetry.polkadot.io/#/Local%20Testnet) . You'll need two terminals windows open.
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
If you want to see the multi-node consensus algorithm in action locally, then you can create a local testnet with two validator nodes for Alice and Bob, who are the initial authorities of the genesis chain that have been endowed with testnet units. Give each node a name and expose them so they are listed on the Polkadot [telemetry site](https://telemetry.polkadot.io/#/Local%20Testnet). You'll need two terminal windows open.
```

File: node-template/README.md [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+
+We'll start Alice's substrate node first on default TCP port 30333 with her chain database stored locally at `/tmp/alice`. The Bootnode ID of her node is `QmQZ8TjTqeDj3ciwr93EJ95hxfDsb9pEYDizUAbWpigtQN`, which is generated from the `--node-key` value that we specify below:
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
We'll start Alice's substrate node first on default TCP port 30333 with her chain database stored locally at `/tmp/alice`. The bootnode ID of her node is `QmQZ8TjTqeDj3ciwr93EJ95hxfDsb9pEYDizUAbWpigtQN`, which is generated from the `--node-key` value that we specify below:
```

File: node-template/README.md [hunk 6]
```diff
@@ -1,3 +1,68 @@
...
+cargo run --release -- \
+  --base-path /tmp/alice \
+  --chain=local \
+  --alice \
+  --node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+  --telemetry-url ws://telemetry.polkadot.io:1024 \
+  --validator
+```
+
+In the second terminal, we'll run the following to start Bob's substrate node on a different TCP port of 30334, and with his chain database stored locally at `/tmp/bob`. We'll specify a value for the `--bootnodes` option that will connect his node to Alice's Bootnode ID on TCP port 30333:
```
Comment (joepetrowski) [inline node-template/README.md]:
```suggestion
In the second terminal, we'll start Bob's substrate node on a different TCP port of 30334, and with his chain database stored locally at `/tmp/bob`. We'll specify a value for the `--bootnodes` option that will connect his node to Alice's bootnode ID on TCP port 30333:
```

**Comment by xlc** [comment]
@joepetrowski The second part of the readme is copied from the main README.adoc file so you may want to apply your suggestions to it as well.

**Comment by joepetrowski** [comment]
> @joepetrowski The second part of the readme is copied from the main README.adoc file so you may want to apply your suggestions to it as well.

Thanks, will update

**Comment by joepetrowski** [comment]
Also, since the README here suggests to run in `--release` mode, but the `cargo build` step does not include that flag, it will need to recompile. For playing around with template, maybe it's fine to run without it. I don't know the performance difference enough to recommend one way or the other, but the build should be consistent with the run.

**Comment by xlc** [comment]
I personally find the debug build performance is enough for development purpose and almost always use debug build during development and basic testing. I will remove `--release` flag unless anyone oppose.
The commands are intended for development / testing purpose anyway.

**Pull request merged**